kotlintest-extensions/src/test/kotlin/io/kotlintest/extensions/junitxml/JUnitXmlTest.kt

```diff
@@ -1,6 +1,5 @@
-package io.kotlintest.extensions.allure
+package io.kotlintest.extensions.junitxml
```

Comment:
Same

Reply:
Is junitxml related to allure in any way? I moved it to the a different package because it was a different thing

Reply:
No, the junit xml extension can in theory be deleted. Gradle and maven do the same thing.

Reply:
What was the original reason for it to be created?
Is there any chance that anybody is using it? I'm for deleting it

Reply:
Ok, I removed it.